src/ess/snspowder/powgen/workflow.py

```diff
@@ -13,6 +13,25 @@ def default_parameters() -> dict:
     return {NeXusDetectorName: "powgen_detector"}
 
 
+def dummy_compute_detector_time_of_flight(
+    detector_data: DetectorData[RunType],
+) -> TofData[RunType]:
+    """
+    Dummy function to compute time-of-flight data from detector data.
+    The Powgen data already contains a `tof` coordinate.
+    We also do not have the chopper information to compute a better estimate.
+
+    Parameters
+    ----------
+    detector_data:
+        Data from the detector.
+    """
+    return TofData[RunType](detector_data)
```

Comment:
Can we avoid this dummy provider by changing `extract_raw_data` to return `TofData` instead of `DetectorData`?